core/shared/src/main/scala/cats/effect/Concurrent.scala

```diff
@@ -353,26 +354,67 @@ object Concurrent {
     }
 
   /**
-    * Lazily memoizes `f`. For every time the returned `F[F[A]]` is
-    * bound, the effect `f` will be performed at most once (when the
-    * inner `F[A]` is bound the first time).
+    * Lazily memoizes `f`. Assuming no cancelation happens, the effect
```

Comment:
Standardized spelling

```suggestion
    * Lazily memoizes `f`. Assuming no cancellation happens, the effect
```

Reply:
*twitches*

Reply:
:us: :fireworks: :statue_of_liberty: